.travis.yml

```diff
@@ -29,13 +11,13 @@ language: php
 env:
   global:
     - DEFAULT_COMPOSER_FLAGS="--prefer-dist --no-interaction --no-progress --optimize-autoloader"
-    - TASK_TESTS_PHP=1
-    - TASK_TESTS_COVERAGE=0
     - TRAVIS_SECOND_USER=travis_two
 
 
 services:
+  - memcached
   - mysql
+  - postgresql
```

Comment:
Why do we need postgresql for MySQL package?

Reply:
Done.